LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 stampery-labs
```

Comment:
Let's do:
```suggestion
Copyright (c) 2021 Witnet Foundation and contributors
```

Reply:
Lol, it was copied from another repo...I just find out where it came from